session6/src/player.py

```diff
@@ -0,0 +1,72 @@
+import pygame
+import os
+
+
+GRAVITY = 0.25
+JUMP_VELOCITY = -1 * 12
+
+
+class Player(pygame.sprite.Sprite):
+    def __init__(self):
+        super().__init__()
+        self.x_speed = 5  # speed in the x direction
+        self.y_speed = 0  # speed in the y direction
+        image_location = os.path.join(
+            "assets", "player.png"
+        )  # find the path of the player image
```

Comment:
Same as above. Also personally prefer capitalized first letters in comments
```suggestion
        image_location = os.path.join("assets", "player.png")  # Find the path of the player image
```